README.md

```diff
@@ -3,6 +3,10 @@
 This app allows your organization to trace Github Actions with Sentry. You can use this to get insights of what parts of your CI are slow or failing often.
 It works by listening to a Github workflow events via a webhook in your repository. These events are the stored in Sentry as performance transactions.
 
+`github_sdk.py` has the generic logic to submit Github jobs as transactions. Eventually this file could be released separatedly.
+`handle_event.py`: Business logic goes here.
```

Comment:
I've taken some code from main.py and placed it in this file in order to test it.